setup_sklearnex.py

```diff
@@ -37,8 +37,11 @@
 
 # sklearnex setup
 setup(name="scikit-learn-intelex",
-      description="Extension for scikit-learn with helps \
-            Intel(R) oneAPI Data Analytics Library",
+      description="Intel(R) Extension for Scikit-learn speeds up scikit-learn beyond by "
```

Comment:
"Intel(R) Extension for Scikit-learn*" since it could be the first usage of full name in the code.